src/components/inputs/rich-text-input/editor.js

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import Button from './button';
+import StyleDropdown from './style-dropdown';
+import MarkButton from './buttons/mark-button';
+import filterDataAttributes from '../../../utils/filter-data-attributes';
+import useToggleState from '../../../hooks/use-toggle-state';
+import {
+  BoldIcon,
+  ItalicIcon,
+  UnorderedListIcon,
+  OrderedListIcon,
+  UnderlineIcon,
+  RedoIcon,
+} from './icons';
+import { RevertIcon } from '../../icons';
```

Comment:
I've noticed the `RevertIcon` has a different size from `RedoIcon` in the UI, and it's because of the SVGs (probably because they were done in different instances / by different designers).

This is kind of a nitpick but just making sure it isn't forgotten -- design should provide all the icons with the same sizes and crops.